Gauntlet.toml

```diff
@@ -3971,6 +3981,51 @@ document = "chanzuckerberg/czid-workflows:/workflows/short-read-mngs/host_filter
+permalink = "https://github.com/chanzuckerberg/czid-workflows/blob/8d57beab9fd6ab640826af0fdc6bff9b22d95163/workflows/short-read-mngs/local_driver.wdl/#L23"
+
+[[diagnostics]]
+document = "chanzuckerberg/czid-workflows:/workflows/short-read-mngs/local_driver.wdl"
+message = "local_driver.wdl:23:17: warning: missing required call input `hisat2_index_tar` for workflow `czid_host_filter`"
+permalink = "https://github.com/chanzuckerberg/czid-workflows/blob/8d57beab9fd6ab640826af0fdc6bff9b22d95163/workflows/short-read-mngs/local_driver.wdl/#L23"
+
+[[diagnostics]]
+document = "chanzuckerberg/czid-workflows:/workflows/short-read-mngs/local_driver.wdl"
+message = "local_driver.wdl:23:17: warning: missing required call input `host_genome` for workflow `czid_host_filter`"
+permalink = "https://github.com/chanzuckerberg/czid-workflows/blob/8d57beab9fd6ab640826af0fdc6bff9b22d95163/workflows/short-read-mngs/local_driver.wdl/#L23"
+
+[[diagnostics]]
+document = "chanzuckerberg/czid-workflows:/workflows/short-read-mngs/local_driver.wdl"
+message = "local_driver.wdl:23:17: warning: missing required call input `human_bowtie2_index_tar` for workflow `czid_host_filter`"
+permalink = "https://github.com/chanzuckerberg/czid-workflows/blob/8d57beab9fd6ab640826af0fdc6bff9b22d95163/workflows/short-read-mngs/local_driver.wdl/#L23"
+
+[[diagnostics]]
+document = "chanzuckerberg/czid-workflows:/workflows/short-read-mngs/local_driver.wdl"
+message = "local_driver.wdl:23:17: warning: missing required call input `human_hisat2_index_tar` for workflow `czid_host_filter`"
+permalink = "https://github.com/chanzuckerberg/czid-workflows/blob/8d57beab9fd6ab640826af0fdc6bff9b22d95163/workflows/short-read-mngs/local_driver.wdl/#L23"
+
+[[diagnostics]]
+document = "chanzuckerberg/czid-workflows:/workflows/short-read-mngs/local_driver.wdl"
+message = "local_driver.wdl:23:17: warning: missing required call input `kallisto_idx` for workflow `czid_host_filter`"
+permalink = "https://github.com/chanzuckerberg/czid-workflows/blob/8d57beab9fd6ab640826af0fdc6bff9b22d95163/workflows/short-read-mngs/local_driver.wdl/#L23"
+
+[[diagnostics]]
+document = "chanzuckerberg/czid-workflows:/workflows/short-read-mngs/local_driver.wdl"
+message = "local_driver.wdl:23:17: warning: missing required call input `max_input_fragments` for workflow `czid_host_filter`"
+permalink = "https://github.com/chanzuckerberg/czid-workflows/blob/8d57beab9fd6ab640826af0fdc6bff9b22d95163/workflows/short-read-mngs/local_driver.wdl/#L23"
+
+[[diagnostics]]
+document = "chanzuckerberg/czid-workflows:/workflows/short-read-mngs/local_driver.wdl"
+message = "local_driver.wdl:23:17: warning: missing required call input `max_subsample_fragments` for workflow `czid_host_filter`"
+permalink = "https://github.com/chanzuckerberg/czid-workflows/blob/8d57beab9fd6ab640826af0fdc6bff9b22d95163/workflows/short-read-mngs/local_driver.wdl/#L23"
+
```

Comment:
These are also real. They're specified in the YAML here: https://github.com/chanzuckerberg/czid-workflows/blob/8d57beab9fd6ab640826af0fdc6bff9b22d95163/workflows/short-read-mngs/host_filter_defaults.yml. @a-frantz - Interestingly, it looks like `miniwdl` supports YAML as an input file, which appears to be undocumented behavior.

Reply:
> Interestingly, it looks like `miniwdl` supports YAML as an input file, which appears to be undocumented behavior.

I had no idea this was the case, but is good to know!